react/AppTile/Readme.md

```diff
@@ -2,9 +2,9 @@ Component used to show an app status, can be used in
 a list of apps for example in the Store.
 
 ```jsx
-import AppTile from '.'
-import mockApps from '../AppSections/_mockApps'
-import { I18n } from '../I18n'
+import AppTile from 'cozy-ui/transpiled/react/AppTile'
+import mockApps from 'cozy-ui/transpiled/react/AppSections/_mockApps'
```

Comment:
This one is not needed as _mockApps is not part of the public API

Reply:
nit: next time, use Github suggestion 🙏

Reply:
Sadly, the suggestion icon is not accessible when commenting from the "commits" tab :(